src/frontend/get-github-configuration.ts

```diff
@@ -65,21 +71,37 @@ async function getInstallationsWithAdmin(installations: Octokit.AppsListInstalla
 			type: installation.target_type
 		});
 
-		const authedApp = await app.auth(installation.id);
-		enhanceOctokit(authedApp);
+		if(await booleanFlag(BooleanFlags.USE_NEW_GITHUB_CLIENT_TO_COUNT_REPOS, true, jiraHost)){
+			const githubClient = new GitHubClient(installation.id, log);
```

Comment:
I've changed it a bit. Instead of creating a new logger take it from the req.log